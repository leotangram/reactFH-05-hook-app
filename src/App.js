import React from 'react'
import UseState from './components/01-useState/UseState'
import UseEffect from './components/02-useEffect/UseEffect'
import Examples from './components/03-examples/Examples'
import UseRef from './components/04-useRef/UseRef'

const App = () => {
  return (
    <>
      <UseState />
      <UseEffect />
      <Examples />
      <UseRef />
    </>
  )
}

export default App
