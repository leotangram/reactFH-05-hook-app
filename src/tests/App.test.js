import React from 'react'
import { shallow } from 'enzyme'
import App from '../App'

describe('<App />', () => {
  test('should show correctly', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
  })
})
