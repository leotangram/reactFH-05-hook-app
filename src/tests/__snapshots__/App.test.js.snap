// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should show correctly 1`] = `
<Fragment>
  <UseState />
  <UseEffect />
  <Examples />
  <UseRef />
  <UseLayoutEffect />
  <Memos />
  <Padre />
  <TodoApp />
  <br />
  <br />
  <br />
  <MainApp />
</Fragment>
`;
